packages/nuxt-module/lib/module.js

```diff
@@ -2,12 +2,17 @@
 const consola = require('consola')
 const path = require('path')
 
+const rawSourcePackages = {
```

Comment:
to w tamtego ifa mozna wrzucic, zeby ladnie poukladane bylo, pozniej to pewnie i tak przeniose w jakies inne pliki